src/locales/locale-en.json

```diff
@@ -103,7 +103,8 @@
       "DOWNLOAD_BUTTON_LABEL": "Download ICS",
       "SUBSCRIBE_TITLE": "Subscribe",
       "SUBSCRIBE_TEXT": "Use this URL to subscribe as an ICS calendar:",
-      "MORE_INFOS": "More info about ICS"
+      "MORE_INFOS": "More info about ICS",
+      "REFRESH_TOKEN": "Everyone who knows your secret URL can read your activities. To avoid this, please don't share the URL with other people. If this happenend to you, click here to create a fresh secret URL and invalidate previous URLs:"
```

Comment:
Minor language point, instead of _can **read** your activities_ would use _see_, _view_, or _access_ instead.

It also seems a bit wordy, so maybe a total rewrite (borrowing a bit from [googles dialog](https://www.varonis.com/blog/the-dangers-of-shared-links) for shared links):

_Anyone who has the secret link can access. No sign-in required._

Then maybe have the button as _Reset Secret Link_

Just an idea...